code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+import ClimbingBoxLoader from "react-spinners/ClimbingBoxLoader"
+
+import { BackButton } from '../components/BackButton'
+
+export const MovieDetails = () => {
+  const [details, setDetails] = useState([])
+  const {movie_id} = useParams()
+  const [loading, setLoading] = useState(true) //to show the loading spinner by default
+
+  useEffect(() => {
+    fetch(`https://api.themoviedb.org/3/movie/${movie_id}?api_key=2fbbfc5b76142a312e36c1069a8f5912&language=en-US`)
+      .then((res) => res.json())
+      .then((receivedMovieDetails) => {
+        if (receivedMovieDetails.lenght !== 0) { //hiding loading spinner and showing movie details if data is present
+          setDetails(receivedMovieDetails)
+          setLoading(false)
+        }
+      })
+      .catch(err => console.error(err))
```

Comment:
Excellent that you use consoles while coding, however, it's good to remove it before deploying.

Reply:
for sure.